Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "a2"
-version = "0.7.0"
+version = "0.8.0"
```

```suggestion
version = "0.7.0"
```

I'll bump the version in a separate PR as there is another open

---

How about split the difference and I submit this as `0.7.1`?

---

Sounds good to me! Thanks as always